File: ts-sdk/src/sdk/models/shared/confirmholdrequest.ts
import { SpeakeasyBase, SpeakeasyMetadata } from "../../../internal/utils";


export class ConfirmHoldRequest extends SpeakeasyBase {
  @SpeakeasyMetadata({ data: "json, name=amount" })
  amount?: number;

  @SpeakeasyMetadata({ data: "json, name=final" })
  final?: boolean;
}

File: java-sdk/lib/src/main/java/dev/formance/javasdk/models/operations/ConfirmHoldRequest.java
package dev.formance.javasdk.models.operations;

import dev.formance.javasdk.utils.SpeakeasyMetadata;

public class ConfirmHoldRequest {
    public ConfirmHoldPathParams pathParams;
    public ConfirmHoldRequest withPathParams(ConfirmHoldPathParams pathParams) {
        this.pathParams = pathParams;
        return this;
    }
    @SpeakeasyMetadata("request:mediaType=application/json")
    public dev.formance.javasdk.models.shared.ConfirmHoldRequest request;
    public ConfirmHoldRequest withRequest(dev.formance.javasdk.models.shared.ConfirmHoldRequest request) {
        this.request = request;
        return this;
    }
}
